xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -85,14 +85,14 @@ struct FirstRunView: View {
                             send: app.send,
                             tag: AppAction.inviteCodeFormField
                         ),
-                        caption: Group {
+                        caption: .text(Func.run {
                             switch app.state.inviteCodeRedemptionStatus {
                             case .failed(_):
-                                Text("Could not redeem invite code")
+                                return "Could not redeem invite code"
                             case _:
-                                Text("You can find your invite code in your welcome email")
+                                return "You can find your invite code in your welcome email"
                             }
-                        },
+                        }),
```

Comment:
We could consider factoring the text out into a view computed property.

We should use `String(localized:)` for the strings.